Matthew_Bae.cfdg

```diff
@@ -0,0 +1,17 @@
+startshape SNOWFLAKE
+
+
+rule SNOWFLAKE {
+	CIRCLE { size 0.17 }
+	NEXTLAYER { size .55}
+}
+
+rule NEXTLAYER {
+    4* [r 90] CIRCLE { r 45 x 0.5 s 0.15 }
```

Comment:
I don't believe that the r 45 inside this circle has any effect.  Copied the program and changed it and I don't see a difference.